Sources/SwiftGen/Config/Config+Run.swift

```diff
@@ -10,12 +10,12 @@ import StencilSwiftKit
 import SwiftGenKit
 
 extension Config {
-  func runActions(verbose: Bool) throws {
+  func runActions(verbose: Bool, logger: @escaping (LogLevel, String) -> Void = logMessage) throws {
```

Comment:
I'm not sure what makes it required to be `@escaping`? I mean yeah it's passed around to the next helper function, but none of the helper/private functions we pass that closure to ends up _capturing_ / _escaping_ the closure, they use it right away, right?

Or is it because of `parallelCompactMap` which implicitly captures it? (if so, I think we could work around it either by using a local variable outside the `parallelCompactMap` referencing the `logger` parameter, or by using `withoutActuallyEscaping` 🤔

Reply:
Yeah it's because of `parallelCompactMap`. Didn't think about those options, will see if I can get those to work.